tests/integration/llm/prepare.py

```diff
@@ -280,22 +280,105 @@
         "option.task": "text-generation",
         "option.model_id": "s3://djl-llm/opt-30b/"
     },
-    "open-llama-13b": {
+    "open-llama-13b-fp16-deepspeed": {
         "option.task": "text-generation",
+        "option.dtype": "fp16",
+        "option.engine": "deepspeed",
         "option.model_id": "s3://djl-llm/open-llama-13b/"
     },
-    "gpt-j-6b": {
+    "open-llama-13b-fp16-huggingface": {
+        "option.task": "text-generation",
+        "option.dtype": "fp16",
+        "option.engine": "huggingface",
+        "option.model_id": "s3://djl-llm/open-llama-13b/"
+    },
+    "open-llama-13b-bf16-deepspeed": {
+        "option.task": "text-generation",
+        "option.dtype": "bf16",
+        "option.engine": "deepspeed",
+        "option.model_id": "s3://djl-llm/open-llama-13b/"
+    },
+    "open-llama-13b-bf16-huggingface": {
+        "option.task": "text-generation",
+        "option.dtype": "bf16",
+        "option.engine": "huggingface",
+        "option.model_id": "s3://djl-llm/open-llama-13b/"
+    },
+    "open-llama-13b-smoothquant": {
+        "option.task": "text-generation",
+        "option.model_id": "TheBloke/Llama-2-13B-fp16",
```

Comment:
Is this the correct model I should use for smoothquant benchmark?

Reply:
We will want to move it over to an s3 bucket to minimize container startup, but I think we want apples to apples comparisons, so this should be the s3 llama model used by the normal deepspeed performance test.

Reply:
Got it. Let me update it.